tcms/testruns/migrations/0004_squashed.py

```diff
@@ -60,7 +60,7 @@ class Migration(migrations.Migration):
                 ("case_run_id", models.AutoField(primary_key=True, serialize=False)),
                 ("case_text_version", models.IntegerField()),
                 ("running_date", models.DateTimeField(blank=True, null=True)),
-                ("close_date", models.DateTimeField(blank=True, null=True)),
+                ("stop_date", models.DateTimeField(blank=True, null=True)),
```

Comment:
Undo the changes to this file. Historical migrations generally do not need to be modified.

You can read more about migrations at https://docs.djangoproject.com/en/3.1/topics/migrations/

Reply:
Still unresolved.

Reply:
I changed the names of "stop_date" back to "close_date" as they were written before. I'm not really sure what else I should change since thats the only thing that was changed in this historic file 👀

Reply:
> I manually changed the names of "stop_date" back to "close_date" as they were written before. I'm not really sure what else I should change since thats the only thing that was changed in this historic file

Well if that's the only thing which changed and I tell you that it is not supposed to be there then when you update the file for a second time it should look exactly like it was before.

> Do I have to undo the change to this file using the git commands?

I don't follow what you mean but in any case all changes made locally should be committed and pushed to GitHub.